internal/dashboard/router/v1/user_router.go

```diff
@@ -21,4 +21,5 @@ import (
 func InitUserRoutes(g *gin.RouterGroup) {
 	g.POST("/login", h.Wrap(h.Login))
 	g.POST("/logout", h.Wrap(h.Logout))
+	g.POST("/auth/:token", h.Wrap(h.Authz))
```

Comment:
Why not just use Auth

Reply:
`Auth` is confusing that could stand for Authentication and Authorization, while `Authz` is the latter.